### PR TITLE
Config use dictionary

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
Ref #261

**Comment by aashish24** [comment]
The goal of this PR to enable script to define server configuration within the script / code self eliminating the need for a config file on the disk.

**Comment by aashish24** [comment]
@alculquicondor please review.

File: tests/utils.py [hunk 1]
```diff
@@ -14,22 +14,29 @@
...
+def request(config_content, layer_name, format, row, column, zoom):
     '''
-    Helper method to write config_file_content to disk and do
+    Helper method to write config_file to disk and do
     request
     '''
 
-    absolute_file_name = create_temp_file(config_file_content)
-    config = parseConfigfile(absolute_file_name)
+    is_string = isinstance(config_content, str)
```
Comment (alculquicondor) [inline tests/utils.py]: use `basestring`

**Comment by aashish24** [comment]
thansk @alculquicondor

**Pull request merged**